TutorLizard.Web/Controllers/StudentController.cs

```diff
@@ -61,16 +61,11 @@ public IActionResult AdRequests(IFormCollection buttonAction)
 
             StudentsAdRequestsRequest request = new(studentId);
 
-            StudentsAdRequestsResponse response = new()
-            {
-                //data for tests only
-                AdRequests = [new AdRequestsListDto(1, 1, 1, false, "xyz", "yxz", false)]
-            };
+            StudentsAdRequestsResponse response = await _studentService.ViewAdRequests(request);
 
             if (!string.IsNullOrWhiteSpace(buttonAction["btnRequest"]))
```

Comment:
I don't understand this if statement and use of `IFormCollection` in general. This action is a GET, it's not a response to a form being sent.

Reply:
Deleted